{
	"next_topic_id": 2,
	"topic": [
		{
			"topic_id": 1555468864,
			"title": "Errors with Zeronet",
			"body": "Hello, I am having a few problems with Zeronet. Can anybody give me any pointers on what I should do to fix it/ get rid of the notification every time? ZeroNet seems to be running completely fine. Thanks in advance!\n\n`Internal error: ValueError: Expecting property name: line 5 column 3 (char 101)`\n```\nUiWebsocket.py line 83 > UiWebsocket.py line 267 > ContentFilterPlugin.py line 148 > SiteStorage.py line 307 > __init__.py line 291 > __init__.py line 339 > decoder.py line 364 > decoder.py line 380\n```\n\nI also cannot access Zeronet from the Tor Browser. I whitelisted 127.0.0.1 and the page will load with the logo in the top right corner, but nothing else will show up.",
			"added": 1555468863
		}
	],
	"topic_vote": {},
	"next_comment_id": 2,
	"comment": {
		"8_1J3rJ8ecnwH2EPYa6MrgZttBNc61ACFiCj": [
			{
				"comment_id": 1,
				"body": "Hello,\nI keep getting this error every time I run ZeroNet. Can anybody give me any pointers on what I should do to fix it/ get rid of the notification every time? ZeroNet seems to be running completely fine. \n\n`Internal error: ValueError: Expecting property name: line 5 column 3 (char 101)`\n```\nUiWebsocket.py line 83 > UiWebsocket.py line 267 > ContentFilterPlugin.py line 148 > SiteStorage.py line 307 > __init__.py line 291 > __init__.py line 339 > decoder.py line 364 > decoder.py line 380\n```",
				"added": 1555375684
			}
		]
	},
	"comment_vote": {
		"1_15goYtrPd2KrFgP91i8ZH88gbjYfQB7n9L": 1
	}
}